### ISSUE TITLE
Regarding RadialEmbeddingBlock

### ISSUE THREAD

**Issue by swillow** [ACEsuit/mace-jax#6]
Please review the following line in the `RadialEmbeddingBlock` class:

```python
basis = self.basis_functions(lengths, self.r_max)  # [n_edges, num_basis]
```

https://github.com/ACEsuit/mace-jax/blob/4b899de2101c6e2085ee972aeac0e46a334fd9a0/mace_jax/modules/blocks.py#L87

I believe it should be:

```python
basis = self.basis_functions(lengths, bessel_basis.number)  # [n_edges, num_basis]
```

**Comment by mariogeiger** [comment]
This gin configuration is hard to follow, we should have used another config method like for instance hydra but I believe it's correct. Here are all the relevant lines of code to track the radial function throughout the code base.

https://github.com/ACEsuit/mace-jax/blob/4b899de2101c6e2085ee972aeac0e46a334fd9a0/mace_jax/modules/models.py#L99-L104

https://github.com/ACEsuit/mace-jax/blob/4b899de2101c6e2085ee972aeac0e46a334fd9a0/mace_jax/tools/gin_model.py#L189-L198

https://github.com/ACEsuit/mace-jax/blob/4b899de2101c6e2085ee972aeac0e46a334fd9a0/configs/aspirin.gin#L20-L21

https://github.com/ACEsuit/mace-jax/blob/4b899de2101c6e2085ee972aeac0e46a334fd9a0/mace_jax/tools/gin_model.py#L31-L33

**Comment by swillow** [comment]
Thank you for the clarification.
